keps/sig-docs/20191020-doc-policies-for-third-party-content.md

```diff
@@ -0,0 +1,339 @@
+[kubernetes-sigs](https://github.com/kubernetes-sigs) repositories.
+
+## Introduction
+
+Kubernetes documentation teaches Kubernetes users about how
+Kubernetes works, how to use in-project Kubernetes features, and how to
+build on top of Kubernetes infrastucture.
+
+Feature docs are not a place for vendor pitches. Nonetheless, SIG Docs sometimes
+receives pull requests to place advertising-like content on the Kubernetes
+website. Some PRs clearly do not belong in feature docs, but other
+instances are less clear.
+
+Feature docs also contain dual-sourced content. A good practice for code
+project docs is to host single-sourced content only, and to provide
+links to other providers’ single-sourced content. This simplifies
+version management and reduces the work required to maintain content.
+
+This KEP proposes that Kubernetes documentation sets these boundaries:
+
+- Kubernetes documentation includes 
+
+This KEP defines how to handle third-party and dual-sourced content in 
+documentation, so that authors can
+judge what is appropriate to propose and so that PR approvers can make
+consistent, fair decisions during the review process.
+
+## Motivation
+
+SIG Docs publishes Kubernetes
+documentation on kubernetes.io in line with its
+[charter](https://github.com/kubernetes/community/blob/master/sig-docs/charter.md#scope)
+and sets standards for website content. Prior to this KEP, there are no
+clear guidelines or standards for third-party and dual-sourced content.
+
+The Kubernetes documentation is currently a mix of both 1) documentation
+describing the Kubernetes open source project; and 2) content describing
+how to install or use Kubernetes on several third party Kubernetes
+offerings.
+
+Some third party content is necessary in order for Kubernetes to
+function. For example: container runtimes (containerd, CRI-o, Docker), 
+networking policy (CNI plugins), Ingress controllers, and logging all require 
+third party components. Pages like [Logging Using Elasticsearch and Kibana](https://kubernetes.io/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana/) 
+are highly specific to a third party offering and seem more like third party 
+product documentation than Kubernetes open source documentation.
+
+### Goals
+
+The goal of Kubernetes documentation is to accurately document in-project
+functionality for Kubernetes, and to eliminate barriers to effective
+contribution and understanding.
+
+The goal of this KEP is to reach and document a consensus on what
+types of third-party content are appropriate for inclusion in Kubernetes
+documentation; standards for including third-party content; and to create
+consistent policies for docs handle third-party and dual-sourced content.
+
+To address its goal, this KEP focuses on the following issues:
+
+<del>
+
+1. What third party content is appropriate for inclusion in the Kubernetes
+documentation?
+
+Proposed: Third-party content is permitted if it's required for Kubernetes to
```

Comment:
it's -> it is ?

Reply:
"It's" is grammatically correct here; is it just a preference to avoid contractions?